scripts/build.ps1

```diff
@@ -175,8 +179,10 @@ function CMake-Generate {
             "arm"   { $Arguments += "arm" }
             "arm64" { $Arguments += "arm64" }
         }
-    } else {
+    } elseif ($IsLinux) {
         $Arguments += " 'Linux Makefiles'"
+    } elseif ($IsMacOS) {
```

Comment:
I've been trying to keep everything as "Darwin" so it works on macOS + iOS, but making the distinction between macOS and iOS might be useful... unclear

Reply:
This is a variable built into Powershell. Building for iOS would be a cross compilation, which would need to be set up separately.

Reply:
Yeah, let's keep them separate at this level for now. If/when iOS support is added, we can revisit this decision.